sdk/formrecognizer/ai-form-recognizer/samples/javascript/README.md

```diff
@@ -16,6 +16,7 @@ These sample programs show how to use the JavaScript client libraries for Azure
 | [differentiateLabeledUnlabeled.js][differentiatelabeledunlabeled] | See the differences in output when using a custom model trained with labeled data and one trained with unlabeled data |
 | [copyModel.js][copymodel]                                         | Copy a custom model from one Form Recognizer resource to another                                                      |
 | [authenticationMethods.js][authenticationmethods]                 | authenticates a service client using both Azure Active Directory and an API key                                       |
+| [deleteAllModels.js][deleteallmodels]                             | Delete all models in a Form Recognizer account                                                                        |
```

Comment:
curious about this sample since I don't think other languages have it. Is there a reason for adding?

Reply:
Keyvault has one here in JS, and I've found working with this service that I really needed this code on hand to clean up my own FR account. I figured if I needed it, someone else probably will as well.